app/controllers/provider_interface/organisation_permissions_setup_controller.rb

```diff
@@ -77,13 +77,6 @@ def success; end
 
   private
 
-    def require_access_to_manage_provider_relationship_permissions!
```

Comment:
Copied over from the old controller by accident but not used

Reply:
can we update brakeman and remove mentions of this method?